CONTRIBUTING.md

```diff
@@ -244,7 +244,6 @@ If you need a custom C++/Cuda op for your test, compile your ops with
 
 ```bash
 python configure.py
-python configure.py --no-deps   # if you don't want any dependencies installed with pip
 bash tools/install_so_files.sh  # Linux/macos/WSL2
 sh tools/install_so_files.sh    # PowerShell
 ```
```

Comment:
The instructions for running within the docker container yourself probably should be updated to let them know they need to install tensorflow?
```
(addons-nightly) sean-morgan@ubuntu-spmorgan:~/code/tf/seanpmorgan/addons$ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:2.1.0-custom-op-ubuntu16
root@d1c464db8012:/addons# python3
Python 3.6.10 (default, Dec 19 2019, 23:04:32) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'tensorflow'
```

Reply:
This may be less of an issue once we publish a docker developer image